bool palm_reflex_triggered
